lib/src/vDraw_skin.cpp

```diff
@@ -0,0 +1,349 @@
+/*
+ *   Copyright (C) 2017 Event-driven Perception for Robotics
```

Comment:
Copyright year to be updated.
```suggestion
 *   Copyright (C) 2020 Event-driven Perception for Robotics
```